apps/mobile-alphabet/src/app/Components/alphabet-card-detail.screen.tsx

```diff
@@ -80,84 +92,58 @@ export function AlphabetCardDetailScreen({ route }) {
         standalone_image,
     } = selectedCard;
 
+    const swipeRight = () => {
+        setSelectedLetterSequenceNumber(
+            ((selectedLetterSequenceNumber - 2 + alphabetCards.length) % alphabetCards.length) + 1
```

Comment:
I'm trying to remember where the -2 comes from. Did you run your Detox test?

Reply:
We refactored this.